home/dot_config/gitomatic/config.yml.tmpl

```diff
@@ -0,0 +1,31 @@
+---
+### Supported input fields
+#  author: Author name for git commits (default "gitomatic")
+#  email: Email address for git commits (default "gitomatic@fribbledom.com")
+#  interval: How often to check for changes (default "1m")
+#  path: Location where the repository exists locally
+#  privkey: Location of private key used for authentication (default "~/.ssh/id_rsa")
+#  pull: Automatically pull changes (default true)
+#  push: Automatically push changes (default true)
+#  username: Username used for authentication
+#  passwordTokenFile: Install Doctor Secrets file containing the Token (encrypted) to be used for authentication. Needed when `username` is provided
+#
+#  ## Example:
+#  repositories:
+#    - name: Repo1
+#      path: "$HOME/workspace/repo1"
+#      pull: true
+#      push: false
+#      interval: 10m
+#      username: developer1@megabyte.space
+#      passwordTokenFile: "GITHUB_TOKEN"
+#    - name: Repo2
```

Comment:
For this file, can we loop through an array in `.chezmoi.yaml.tmpl` where each member of the array has the `path` and `git` address. Then, for the name just put the git address, author use the users name in `.chezmoi.yaml.tmpl`, email use the users email in `.chezmoi.yaml.tmpl`, omit pull/push/interval, omit privkey/username/passwordtokenfile as well.

The gits should be cloned to the specified paths in the above script as well.

Reply:
This is done.

Cloning the repositories is added to the `after_` script to avoid attempting to clone every time the service is started.